docs/en_US/TrainingService/PaiMode.md

```diff
@@ -88,6 +93,8 @@ paiConfig:
 
 Note: You should set `trainingServicePlatform: pai` in NNI config YAML file if you want to start experiment in pai mode. The host field in configuration file is PAI's job submission page uri, like `10.10.5.1`, the default http protocol in NNI is `http`, if your PAI's cluster enabled https, please use the uri in `https://10.10.5.1` format.
 
+
+
 ### Trial configurations
 
 Compared with [LocalMode](LocalMode.md) and [RemoteMachineMode](RemoteMachineMode.md), `trial` configuration in pai mode have these additional keys:
```

Comment:
->  `trial` configuration **on** pai mode **has** additional keys
I think `have these additional keys ` is too chinese english...
@QuanluZhang It's right?

Reply:
"have these additional keys" -> "has the following additional keys"